### PR TITLE
PRO-4597 Fix variants

### PR DESCRIPTION
**Pull request by boutell**
See changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.0.0-beta.3 (2023-08-03)
```
Comment (boutell) [inline CHANGELOG.md:3]: Yes I know, living dangerously, if we don't ship tomorrow I'll update it

File: modules/@apostrophecms/ai-helper-image/index.js [hunk 2]
```diff
@@ -43,8 +45,8 @@ module.exports = {
       async aiHelperFetchImage(_id, url) {
         const response = await fetch(url);
         const buffer = await response.buffer();
-        const temp = path.join(self.apos.rootDir, `data/temp/${_id}.jpg`);
-        await sharp(buffer).toFile(temp);
+        const temp = path.join(self.apos.rootDir, `data/temp/${_id}.png`);
+        await writeFile(temp, buffer);
```
Comment (boutell) [inline modules/@apostrophecms/ai-helper-image/index.js:49]: Not time to JPEG it yet.

File: modules/@apostrophecms/ai-helper-image/index.js [hunk 3]
```diff
@@ -190,11 +192,14 @@ module.exports = {
           const { url, prompt } = image;
           // apos.http has a bug with binary data, use node-fetch
           let temp;
+          let tempJpg;
           try {
             temp = await self.aiHelperFetchImage(_id, url);
+            const tempJpg = temp.replace(/\.\w+$/, '') + '.jpg';
+            await sharp(temp).toFile(tempJpg);
```
Comment (boutell) [inline modules/@apostrophecms/ai-helper-image/index.js:199]: Do it when actually inserting into our media library, leave it PNG until then

**Comment by boutell** [comment]
A PR is in review for this.

**Review by BoDonkey — APPROVED** [review]
Tested. Can generate dancing cat variants and converts to jpeg when saved to the database.

**Pull request merged**